code/modules/surgery/surgery_step.dm

```diff
@@ -132,7 +132,15 @@
 /datum/surgery_step/proc/play_preop_sound(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	if(!preop_sound)
 		return
-	playsound(get_turf(target), preop_sound, 75, TRUE, falloff_exponent = 12, falloff_distance = 1)
+	if(islist(preop_sound))	
+		var/sound_file_use
+		for(var/typepath in preop_sound)
+			if(ispath(tool.type, typepath))
+				sound_file_use = preop_sound[typepath]	
+				return sound_file_use
+		playsound(get_turf(target), sound_file_use, 75, TRUE, falloff_exponent = 12, falloff_distance = 1)
+	else
+		playsound(get_turf(target), preop_sound, 75, TRUE, falloff_exponent = 12, falloff_distance = 1)
```

Comment:
Instead of copy+pasting the same playsound:
- Have 1 playsound at the end of the proc
- declare `var/sound_file_use` outside of the `if` statement you currently have
- set `sound_fine_use` to the selected file if it's a list, or the only set file if it's not